.github/workflows/run-integration-tests-basic.yml

```diff
@@ -0,0 +1,77 @@
+name: Minikube Integration Tests - basic
+on:
+   pull_request:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   push:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   workflow_dispatch:
+
+jobs:
+   run-integration-test:
+      name: Run Minikube Integration Tests
+      runs-on: ubuntu-latest
```

Comment:
I don't believe we can since different strategies call different combinations of parameters, and at different stages in each test, based on the parameters we're calling each test with, we call the python test script at different frequencies and with different parameters.

I might be missing something though, what were you thinking?